Client.xcodeproj/project.pbxproj

```diff
@@ -10252,7 +10258,10 @@
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
 				COPY_PHASE_STRIP = NO;
+				DEVELOPMENT_TEAM = KL8N8XSYF4;
```

Comment:
Was this committed by accident?

Reply:
It's our dev account, i can check if removing it changes anything